extensions/adobe/experience/decisioning/proposition-detail.schema.json

```diff
@@ -16,19 +17,24 @@
     "proposition-detail-datatype": {
       "properties": {
         "xdm:activity": {
-          "title": "Decision Activity",
+          "title": "Decision Activity Snapshot",
           "description": "Snapshot of the activity for which the proposition was made. The object contains at least the xdm:id property value which is the identifier of the decision activity.",
           "$ref": "https://ns.adobe.com/experience/decisioning/activity-detail"
         },
         "xdm:channel": {
```

Comment:
May want to add "meta:status": "deprecated" to this field

Reply:
Thank you, added this meta field